R/Spectra.R

```diff
@@ -552,6 +552,15 @@ NULL
 #'   the comparison of `x[2]` with `y[3]`). If `SIMPLIFY = TRUE` the `matrix`
 #'   is *simplified* to a `numeric` if length of `x` or `y` is one.
 #'
+#' - `deisotopeSpectra`: *deisotope* each spectrum keeping only the monoisotopic
+#'   peak for groups of isotopologues. Isotopologues are estimated using the
+#'   [isotopologues()] function from the *MetaboCoreUtils* package. Note that
+#'   the default parameters for isotope prediction/detection have been
+#'   determined using data from the Human Metabolome Database (HMDB) and
```

Comment:
May be add a reference or link to provide more information about *the default parameters for isotope prediction/detection have been determined using data from HMDB*.